podman.md

```diff
@@ -44,10 +44,12 @@ podman exec -it HibernateTestingMariaDB mysql -U hreact -phreact
 
 * Server
 ```
-sudo podman run --rm -it -e LICENSE=accept --privileged=true --name HibernateTestingDB2 -e DBNAME=hreact -e DB2INSTANCE=hreact -e DB2INST1_PASSWORD=hreact -e PERSISTENT_HOME=false -e ARCHIVE_LOGS=false -e AUTOCONFIG=false -p 50000:50000 ibmcom/db2:11.5.0.0a
+podman run --rm -it -e LICENSE=accept --privileged=true --name HibernateTestingDB2 -e DBNAME=hreact -e DB2INSTANCE=hreact -e DB2INST1_PASSWORD=hreact -e PERSISTENT_HOME=false -e ARCHIVE_LOGS=false -e AUTOCONFIG=false -p 50000:50000 ibmcom/db2:11.5.0.0a
 ```
 
 * CLI
 ```
-sudo podman exec -ti HibernateTestingDB2 bash -c "su - hreact -c db2 connect"
+podman exec -ti HibernateTestingDB2 bash -c "su - hreact -c"
+db2
+connect to hreact
```

Comment:
why 3 steps?

I had already patched the command to connect directly - in a single step

Reply:
Sorry, I will revert it